crates/router/src/connector/adyen/transformers.rs

```diff
@@ -1726,16 +1734,22 @@ fn get_additional_data(item: &types::PaymentsAuthorizeRouterData) -> Option<Addi
     } else {
         None
     };
-    Some(AdditionalData {
-        authorisation_type,
-        manual_capture,
-        execute_three_d,
-        network_tx_reference: None,
-        recurring_detail_reference: None,
-        recurring_shopper_reference: None,
-        recurring_processing_model: None,
-        ..AdditionalData::default()
-    })
+    if authorisation_type.is_none() && manual_capture.is_none() && execute_three_d.is_none() {
```

Comment:
This code handles this:
<img width="1726" alt="Screenshot 2024-12-12 at 14 31 22" src="https://github.com/user-attachments/assets/6e9aa538-3830-467a-a1b5-7d75873481a1" />